### ISSUE TITLE
Polar Laue Mask causing exception when switching back to Raw view mode

### ISSUE THREAD

**Issue by joelvbernier** [HEXRD/hexrdgui#608]
While I do not have the std out and error available right now on Windows, application of a Laue mask to the polar is causing an exception and preventing me from returning to the Raw image mode.  If I remove the mask, it works again.

**Comment by joelvbernier** [comment]
Adding to this: if I am looking a the polar view with a Laue mask applied, and I change, say, the SNIP kernel, the Laue mask doesn't get re-applied.  I need to remove and re-make it.  Sorry but I don't have the terminal output to be more illustrative at the moment (Windows).

**Comment by cjh1** [comment]
This is the exception backtrace:

```python
Traceback (most recent call last):
  File "/home/cjh/work/source/hexrdgui/hexrd/ui/main_window.py", line 683, in change_image_mode
    self.update_all()
  File "/home/cjh/work/source/hexrdgui/hexrd/ui/main_window.py", line 801, in update_all
    line_data = convert_polar_to_raw(data)
  File "/home/cjh/work/source/hexrdgui/hexrd/ui/create_raw_mask.py", line 44, in convert_polar_to_raw
    cart = panel.angles_to_cart(np.radians(line))
  File "/home/cjh/work/source/hexrd/hexrd/instrument.py", line 2174, in angles_to_cart
    angs = np.hstack([tth_eta, np.zeros((len(tth_eta), 1))])
  File "<__array_function__ internals>", line 5, in hstack
  File "/home/cjh/.virtualenvs/hexrd/lib/python3.8/site-packages/numpy/core/shape_base.py", line 346, in hstack
    return _nx.concatenate(arrs, 1)
  File "<__array_function__ internals>", line 5, in concatenate
ValueError: all the input arrays must have same number of dimensions, but the array at index 0 has 3 dimension(s) and the array at index 1 has 2 dimension(s)
```

**Comment by bnmajor** [comment]
#610 provides a temporary fix for the exception by not masking Laue spots in the raw view for now.

With regards to the updated Laue mask - the mask is just created from the current ranges when it is applied and there are no signals in place that indicate when a change has been made, so the old mask must be removed and a new one created. This is something that could be changed at some point, but I don't think before the demo.

**Comment by joelvbernier** [comment]
Looks like a simple dimensionality problem in the input.  It's complaining that the arg `line` is 3-d;  the `angles_to_cart` method is expecting a vstacked 2-d array of (tth, eta) coordinates.  Looks like some reshaping would solve this.  I'll have to fire up the Linux version, because I don't know how to access the debugger through a WIndows PowerShell...

**Comment by cjh1** [comment]
Fixed by #610
